mmv1/products/healthcare/api.yaml

```diff
@@ -307,7 +307,58 @@ objects:
               project. service-PROJECT_NUMBER@gcp-sa-healthcare.iam.gserviceaccount.com must have publisher permissions on the given
               Cloud Pub/Sub topic. Not having adequate permissions will cause the calls that send notifications to fail.
             required: true
-
+      - !ruby/object:Api::Type::NestedObject
+        name: validationConfig
```

Comment:
Can we add a description here as well?
`Configuration for how to validate incoming FHIR resources against configured profiles.`